<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KDCharge</class>
 <widget class="QMainWindow" name="KDCharge">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>732</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1095</width>
    <height>732</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>A5. KD Charges by Customer</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>1070</width>
        <height>659</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>1071</width>
           <height>110</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="1" rowspan="2">
           <widget class="QFrame" name="frame_7">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="typeRadio">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Type:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="tonnageBtn">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Tonnage</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="speciesBtn">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Species</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="woodRadio">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Wood Type:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="forestBtn">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Forest</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="rubberBtn">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Rubber</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" rowspan="2">
           <widget class="QFrame" name="frame_12">
            <property name="minimumSize">
             <size>
              <width>680</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QFrame" name="frame_9">
                  <property name="minimumSize">
                   <size>
                    <width>243</width>
                    <height>52</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>243</width>
                    <height>52</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Customer ID:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="customerID">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_10">
                  <property name="minimumSize">
                   <size>
                    <width>319</width>
                    <height>52</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>319</width>
                    <height>52</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Size:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="length">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="label_3">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>x</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="width">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_11">
                  <property name="minimumSize">
                   <size>
                    <width>180</width>
                    <height>52</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>180</width>
                    <height>52</height>
                   </size>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label_4">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Price:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLineEdit" name="price">
                     <property name="font">
                      <font>
                       <pointsize>11</pointsize>
                      </font>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item alignment="Qt::AlignHCenter">
              <widget class="QFrame" name="frame_8">
               <property name="minimumSize">
                <size>
                 <width>767</width>
                 <height>52</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>767</width>
                 <height>52</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Species: </string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="species">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>1071</width>
           <height>547</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1071</width>
           <height>547</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>30</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="maximumSize">
             <size>
              <width>934</width>
              <height>515</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QFrame" name="frame_15">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Find</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="searchBar"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="tableView"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QFrame" name="frame_16">
               <property name="minimumSize">
                <size>
                 <width>111</width>
                 <height>241</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>111</width>
                 <height>241</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QPushButton" name="resetButton">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Reset</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="deleteButton">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Delete</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="addButton">
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="saveButton">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Save</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="printButton">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Print</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="generateButton">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Generate</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="closeButton">
                  <property name="font">
                   <font>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Close</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
